var/spack/repos/builtin/packages/r-ergm/package.py

```diff
@@ -12,16 +12,22 @@ class RErgm(RPackage):
        part of the "statnet" suite of packages for network analysis."""
 
     homepage = "http://statnet.org"
-    url      = "https://cran.r-project.org/src/contrib/ergm_3.7.1.tar.gz"
-    list_url = "https://cran.r-project.org/src/contrib/Archive/ergm"
+    url      = "https://cloud.r-project.org/src/contrib/ergm_3.7.1.tar.gz"
+    list_url = "https://cloud.r-project.org/src/contrib/Archive/ergm"
 
+    version('3.10.4', sha256='885f0b1a23c5a2c1947962350cfab66683dfdfd1db173c115e90396d00831f22')
+    version('3.10.1', sha256='a2ac249ff07ba55b3359242f20389a892543b4fff5956d74143d2d41fa6d4beb')
     version('3.7.1', '431ae430c76b2408988f469831d80126')
 
-    depends_on('r-robustbase@0.9-10:', type=('build', 'run'))
-    depends_on('r-coda@0.18-1:', type=('build', 'run'))
-    depends_on('r-trust', type=('build', 'run'))
-    depends_on('r-matrix', type=('build', 'run'))
-    depends_on('r-lpsolve', type=('build', 'run'))
-    depends_on('r-mass', type=('build', 'run'))
+    depends_on('r-robustbase@0.93-5:', type=('build', 'run'))
+    depends_on('r-coda@0.19-2:', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('r-coda@0.19.2:', type=('build', 'run'))
```

Reply:
@adamjstewart Both coda and robustbase, as well as many other R packages, use a '-' in the version string.

Reply:
As far as Spack is concerned, it doesn't make a difference. As long as the CRAN page for coda or the CRAN page for ergm uses a dash, that's fine. The trust dependency below is the only one that is definitely incorrect.

Reply:
Yep; even packages that use a dash are sometimes referred to with a '.', which is confusing. I think they actually redirect to the canonical one in the end.

Reply:
Nothing to do here.